src/react-apps/applications/shared/src/version-control/versionControlHeader.tsx

```diff
@@ -62,6 +65,9 @@ class VersionControlHeader extends React.Component<IVersionControlHeaderProps, I
     const url = `${altinnWindow.location.origin}/designerapi/Repository/RepoStatus?owner=${org}&repository=${service}`;
     get(url).then((result: any) => {
       if (this._isMounted) {
+        this.setState({
+          mergeConflict: result.repositoryStatus === 'MergeConflict',
```

Comment:
There is a key called: `"hasMergeConflict": true/false` now that you can consider to use.

Reply:
Or just write the entire content to state and use the keys when you need it.